stdlib/src/fcntl.rs

```diff
@@ -21,6 +21,12 @@ mod fcntl {
     #[pyattr]
     use libc::{F_GETOWN, F_RDLCK, F_SETOWN, F_UNLCK, F_WRLCK};
 
+    #[cfg(target_vendor = "apple")]
+    use libc::F_NOCACHE;
+
+    #[cfg(target_os = "freebsd")]
+    use libc::{F_DUP2FD, F_DUP2FD_CLOEXEC};
+
```

Comment:
Did you check and make sure these were actually _accessible_ when you run the RustPython executable?

Reply:
with last libc release they are.

Reply:
That wasn't my point. Can you get to the things you added _while you're running the RustPython executable?_

Reply:
ah right sorry